### DISCUSSION TITLE
Dashboard description as a link

### DISCUSSION THREAD

**Discussion by heyrod** [grafana/grafana#43717] [Dashboards, Panels & Visualizations]
Sometimes a dashboard needs a good explanation about what it does, how to use it, or where it gets the data from. The options currently available in Grafana do not provide a good way to implement it. You can either
-  add a text panel at the top of the dashboard, consuming the valuable space.
- include the description into a panel description field, but the info icon in the panel corner cannot be easily spotted and recognized by users,
- or embed an icon with a tooltip through the Links section in the dashboard settings.
There is also a Description field in the dashboard settings, but I cannot figure out where and how it's parsed and where this information is displayed elsewhere.
So, I'm using the last option, which is not perfect, adding an external link with the info icon and providing the description in the tooltip. See an example:
![image](https://user-images.githubusercontent.com/39294778/148237683-bac46947-e2cc-4032-b4ba-3e05aba0c3cc.png)
It would be nice to have an option to provide such descriptions on the dashboard directly, without the URL field and with a bigger choice of icons or other visual aids.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
